spec/requests/get_needs_spec.rb

```diff
@@ -0,0 +1,237 @@
+require 'rails_helper'
+
+RSpec.describe 'get info on need', type: :request do
+  describe 'find list of needs' do
+    describe 'allNeeds' do
+      it 'can return all information' do
+        need_1 = Need.create(title: "a need", description: "a test to see if we can test", point_of_contact: "email@gmail.com", start_time: "string value", end_time: "string value", zip_code: "12345", supporters_needed: 12, status: "active")
```

Comment:
On the setup, it may be helpful to use `create!` to ensure we see an error if the `create` fails?